eisbuk-admin/src/store/actions/firestoreOperations/subscriptionHandlers.ts

```diff
@@ -198,7 +200,7 @@ export const subToAttendance: SubscriptionHandler = ({
 }) => {
   const collRef = collection(
     getFirestore(),
-    `${orgPath}/${OrgSubCollection.Attendance}`
+    `${getOrgPath()}/${OrgSubCollection.Attendance}`
```

Comment:
To make it prettier, you can do `collection(getFirestore, getOrgPath(), ...<theRestOfThePathElements>)` since it utilizes the `...rest` operator (sort of like a `path.join()`)

Reply:
Thanks for the suggestion! Done.